### ISSUE TITLE
V2.0: Move to git flow

### ISSUE THREAD

**Issue by Thatkookooguy** [Kibibit/achievibit#174]
###### FEATURE
---
create a develop branch and start moving the [new animations](http://kibibit.io/achievibit-demo) into the application itself to start preparing for an eye candy launch to the github marketplace

I'm hoping to finish the `kibibit-cli` to enforce some git flow rules in an easy way (instead of saying lots of rules in the `CONTRIBUTING.MD` file about git flow, we'll simply ask users to use the cli which will be installed with this repo)

Basically this will help us keep changelogs and update with release versions
